tket/src/serialize/pytket/extension/core.rs

```diff
@@ -40,10 +40,8 @@ impl PytketDecoder for CoreDecoder {
         match (op.op_type, &op.op_box) {
             (PytketOptype::CircBox, Some(OpBox::CircBox { id: _id, circuit })) => {
                 // We have no way to distinguish between input and output bits in the circuit box, so we assume all bits are outputs here.
```

Comment:
Appears that after this PR the correct comment would be:
```suggestion
                // We have no way to distinguish between input and output bits in the circuit box, so we assume all bits are both inputs and outputs here.
```

Right??